tests/Fixer/ClassNotation/FinalInternalClassFixerTest.php

```diff
@@ -526,10 +538,16 @@ public function testFix82(string $expected, ?string $input, array $config): void
      */
     public static function provideFix82Cases(): iterable
     {
-        yield [
+        yield 'readonly attribute' => [
```

Comment:
Maybe too long, but it's hard to name this case properly. Anyway there's no "attribute" here so it's wrong.
```suggestion
        yield 'readonly with enabled `consider_absent_docblock_as_internal_class`' => [
```